src/isomorphic/ReactAsyncWrapper.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactAsyncWrapper
+ */
+
+const ReactBaseClasses = require('ReactBaseClasses');
+
+class ReactAsyncWrapper extends ReactBaseClasses.Component {
```

Comment:
Can we go with very minimal `function` prototype thing rather than Babel class output? We’ve tried very hard to trim down isomorphic React package, class output was a bit bloated last I checked.